common/functions/src/scalars/function_alias.rs

```diff
@@ -42,7 +42,7 @@ impl Function for AliasFunction {
         Ok(args[0].clone())
     }
 
-    fn nullable(&self, _input_schema: &DataSchema) -> Result<bool> {
+    fn nullable(&self, _arg_fields: &[DataField]) -> Result<bool> {
```

Comment:
Most function's nullable should use default as trait.

So it can be removed.

Reply:
> Most function's nullable should use default as trait.
> 
> So it can be removed.

Right, will update the PR